modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,148 @@
+= Migrate Client Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: migrate client applications, migrate applications, migrate 
+
+You might have application client credentials residing outside of Anypoint Platform if you were previously using a different API management platform. You must first migrate those client applications to Anypoint Platform, with their client ID and client secret credentials, before you can start using Anypoint Platform.
+
+Because client IDs are unique in Anypoint Platform, you cannot reuse the same client ID in two different accounts. To avoid the duplication, you must specify unique client IDs for your accounts. 
+
+Your client applications can consume APIs whose organization is registered with an Anypoint Platform client provider or an external client provider. The instructions for these two scenarios differ:
+
+* <<migrate_with_internal,Migrate client applications for APIs with Anypoint Platform client providers>>
+* <<migrate_with_external,Migrate client applications for APIs with external client providers>>
+
+== Prerequisites
+
+Before migrating the client application, ensure that you:
+
+* Obtain the required token and credentials:
+** Client ID and client secret for the client application to migrate
+** Anypoint Platform https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/pages/Authentication/[token]
+* Review the <<considerations,scope and limitations>>
+
+[[considerations]]
+== Scope and Limitations
+
+Before you migrate your client applications to Anypoint Platform, review the following scope and limitations: 
+
+* If the organization of the API that your application consumes has registered an external client provider (for example, PingFederate or Okta), review the section: <<external_client,Organizations with Registered External Client Providers>>.
+* Do not attempt to update the client ID and client secret of your application.
++
+This feature is not supported.
+* Do not use the same user to create more than 500 applications.
+* Do not create applications with duplicate names.
++
+Even if the identity provider (IdP) stores each applications, including duplicate applications, with different client ID and client secret credentials, issues can occur.
+*  Do not attempt to refresh the client ID unless you first delete and then re-register the client application.
+* If the client credentials you provide match the credentials of an existing organization ID on the client provider, the application is associated with that client.
++
+Conversely, if the client credentials that you provide do not match, the request is not associated with the client and fails.
```

Comment:
It is not true that the request fails. If the credentials don't match, it will simply create a client application without associating it to existing clients.

Reply:
Done! Thanks!